src/Tests/Execute/Device.test.ts

```diff
@@ -85,7 +85,7 @@ suite('Device', function() {
       const sw: string = 'TestOS';
       const testSpec = new HostPCSpec(hw, sw);
       try {
-        const hostPC = new Device(deviceName, testSpec);
+        new Device(deviceName, testSpec);
```

Comment:
@llFreetimell I remember that you have fixed some lint warnings about 'not getting return values'. I worry that this change re-arise the warnings. Please check.

Reply:
Interesting.. I think this is same with previous warnings but not caught... @.@

Reply:
https://github.com/Samsung/ONE-vscode/pull/1031
https://github.com/Samsung/ONE-vscode/pull/1034
https://github.com/Samsung/ONE-vscode/pull/1033

I found these related PRs :-D